### PR TITLE
Any% Saves

### PR DESCRIPTION
**Pull request by EJ125**
Implemented practice saves for any% (with and without pg skip), as well as uncommented dev save tool

Saves do not accurately reflect bomb and magic usage at some saves, will need to correct with save specials later, as well as pg skip bedroom state.

### CODE REVIEW

**Review by TakaRikka — COMMENTED** [review]
should probably gitignore the .d / .o files, you dont really want those in the repo

**Comment by EJ125** [comment]
fixed per taka's comments

**Comment by EJ125** [comment]
updated save special callbacks, changed version to 0.2.1

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # tww-gz
 The Wind Waker (SD) ROM hack to practice speedrunning.
 
-Patch your ISO using the romhack-patcher application, which is included in our official releases.
+Patch your ISO (JP) using the romhack-patcher application, which is included in our official releases.
```
Comment (TakaRikka) [inline README.md]: multi version should still support US version, why JP only here?

**Review by TakaRikka — APPROVED** [review]

**Pull request merged**